blueprints/blueprint/files/blueprints/__name__/index.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path')
```

Comment:
Cool thanks.  This is more to just act as like an example page of how to build a blueprint.  Originally I just had everything commented out.  What are you thoughts?  Maybe some more explanation at the top of where to go to find documentation on how to build custom blueprints would be useful?